src/coverlet.msbuild.tasks/coverlet.msbuild.props

```diff
@@ -13,5 +13,8 @@
     <Threshold Condition="$(Threshold) == ''">0</Threshold>
     <ThresholdType Condition="$(ThresholdType) == ''">line,branch,method</ThresholdType>
     <ThresholdStat Condition="$(ThresholdStat) == ''">minimum</ThresholdStat>
+
+    <CoverletBuildTaskPath Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)net472\</CoverletBuildTaskPath>
+    <CoverletBuildTaskPath Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)netcoreapp2.0\</CoverletBuildTaskPath>
```

Comment:
Why not point to the full path here and use it? We do the same for ApiCompat and GenApi, i.e. https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.ApiCompat/build/Microsoft.DotNet.ApiCompat.targets#L9

Reply:
They seem equivalent to me, and the form I used appears in some other packages that generate code during builds (e.g. ANTLR). Deferring a change here as 🚲 🏛

Reply:
I don't really have a strong opinion here I just don't see a reason to introduce a property that points to the msbuild task assembly's directory. The directory doesn't bring any value to the user but a path to the assembly does.